confidential-data-hub/hub/protos/api.proto

```diff
@@ -42,6 +42,15 @@ message ImagePullResponse {
     string manifest_digest = 1;
 }
 
+message InitOverlayNetworkRequest {
+    string pod_name = 1;
+    string lighthouse_pub_ip = 2;
+}
+
+message InitOverlayNetworkResponse {
+    int32 return_code = 1;
+}
+
```

Comment:
I think we probably want to enable this entirely via the cdh config / init-data rather than through the kata agent. You may still want to get the pod name from the agent, though.

Reply:
What is the thing that should actually kick the initialization of the overlay network now, though? The CDH is just a little service that listens for requests. For example, it might receive a request to pull an image. Or, in the case of the overlay network, it would receive a request from the kata-agent to start. If kata-agent isn't doing that, what should? In other words, after we register the service [here](https://github.com/confidential-containers/guest-components/blob/main/confidential-data-hub/hub/src/bin/ttrpc-cdh.rs#L69), who actually makes this API call?